.github/workflows/build_wheels.yml

```diff
@@ -5,15 +5,16 @@ on:
     branches: [main, master, 'release*']
     tags: ['*']
 
-permissions:
-  contents: write
-
 jobs:
   build-wheels:
     if: github.repository == 'python/mypy'
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
       - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
```

Comment:
I'm not _sure_ if the workflow will still correctly operate with this change, because on the last line of this workflow we run `./misc/trigger_wheel_build.sh`, and that script does things such as push git tags:

https://github.com/python/mypy/blob/bac9984a0e2696eed5d65ca49d006c545ba83a54/misc/trigger_wheel_build.sh#L9-L23